### PR TITLE
Showing more user info

### PR DESCRIPTION
**Pull request by rathorevaibhav**
## Summary

Showing preferences and other user data

### CODE REVIEW

**Review by kurund — CHANGES_REQUESTED** [review]
@rathorevaibhav Label is missing. The data is shown correctly.

Screenshot 2020-09-18 at 9 41 18 PM

File: src/containers/Profile/Contact/ContactDescription/ContactDescription.tsx [hunk 1]
```diff
@@ -59,19 +58,29 @@ export const ContactDescription: React.FC<ContactDescriptionProps> = ({
...
+            <div className={styles.DescriptionItemValue}>
+              {Object.keys(settings[key]).filter((settingKey) => {
+                return settings[key][settingKey] == true;
+              }).join(', ')}
+            </div>
+          </div>
+        ))}
+        {typeof fields === "object" && Object.keys(fields).map((key) => (
+          <div key={key}>
+            <div className={styles.DescriptionItem}>{fields[key].label}</div>
```
Comment (kurund) [inline src/containers/Profile/Contact/ContactDescription/ContactDescription.tsx:82]:
```suggestion
            <div className={styles.DescriptionItem}>{fields[key].label ? fields[key].label : key.replace('_', ' ')}</div>
```

**Comment by rathorevaibhav** [comment]
@kurund done with feedbacks. please check.

**Review by kurund — APPROVED** [review]
all good

**Pull request merged**